modules/release-notes/pages/index.adoc

```diff
@@ -124,6 +135,47 @@ vertex function can also be used with vertex input parameters in the query-body
 
 == Fixed issues
 
+=== Fixed and Improved in 3.9.3
+
+==== Crashes and Deadlocks
+* Fixed crash when query thread would keep running after `JobListener` is shutdown (CORE-1730).
+* Fixed GPE crash when using `INSERT` with an `UNKNOWN` accumulator (GLE-5038).
+* Fixed crash when running interpret query with dynamic value (GLE-5600).
+* Fixed JSON Viewer crash on browser when displaying a large response (TOOLS-2283).
+
+==== Functionality
+* Fixed issue of missing files in AdminPortal when downloading an output (TOOLS-2499).
+* Fixed issue primary id could not set a color config for a vertex (TOOLS-2478).
+* Fixed issue where administrator could privilege of vertex `primary_id` in Role Management page of Admin Portal (TOOLS-2430).
+* Fixed issue where error messages of abort requests created large file sizes (CORE-2705).
+* Fixed issue where GPE was still in warmup while running chaos testing (CORE-2670).
+* Fixed issue where user could download query results from the label “View Table Result” without proper privilege (TOOLS-2404).
+* Fixed issue where multiple vertex types return when selecting one vertex type (GLE-5325).
+* Fixed issue of show statement parser error for catalog object created using create statement (GLE-5719).
+* Fixed support for `outdegree()` with a collection of edge types in `INTERPRET` mode (GLE-5476).
+* Fixed issue where delete or edit vertex attributes were incorrect in GST (TOOLS-2258).
```

Comment:
GST is not an accepted user-facing abbreviation.

Reply:
changed with latest commit.